lib/vm/src/vmcontext.rs

```diff
@@ -303,120 +303,77 @@ mod test_vmglobal_import {
     }
 }
 
-/// The fields compiled code needs to access to utilize a WebAssembly linear
```

Comment:
We should probably keep `VMMemoryDefinition` inside of the `vmcontext`

Reply:
done